requirements/docs.txt

```diff
@@ -3,7 +3,7 @@ m2r==0.2.1
 markdown>=3.4.0
 mistune==0.8.4
 myst-parser
--e git+https://github.com/open-mmlab/pytorch_sphinx_theme.git#egg=pytorch_sphinx_theme
+-e git+https://github.com/InternLM/pytorch_sphinx_theme.git#egg=pytorch_sphinx_theme
```

Comment:
how about `sphinx-book-theme` as used in vllm  https://github.com/vllm-project/vllm/blob/9140561059904f133a3c146f04afe13d1cdd5a6c/docs/requirements-docs.txt#L2 ?

URL
https://sphinx-book-theme.readthedocs.io/en/stable/